packages/storybook-utils/src/theme.ts

```diff
@@ -0,0 +1,45 @@
+/** All available Storybook breakpoints / viewports supported by Onyx. */
+export const ONYX_BREAKPOINTS = {
+  "2xsmall": {
```

Comment:
[optional]
```suggestion
  xsmall2 : {
```

would UX also be ok with that? then we wouldn't have to wrap them in "" ^^

Reply:
or xxsmall

Reply:
The breakpoint names are defined by @jannick-ux in Figma: https://www.figma.com/file/YfEUBOHk4J4nYrk04geswG/onyx---Conception-%26-DEV-Exchange?type=design&node-id=14-698&mode=dev

Reply:
The names of the breakpoints have never been used with "" by myself. So having it without those signs is intended.